build.sbt

```diff
@@ -429,12 +465,15 @@ lazy val unidocSettings = Seq(
     "-notimestamp",
     "-quiet"
   ),
-  JavaUnidoc / unidoc / unidocAllSources := (JavaUnidoc / unidoc / unidocAllSources).value.map {
-    sources =>
-      sources.filterNot { source =>
-        source.toString.contains("$") || source.toString.contains("packageprivate")
+  JavaUnidoc / unidoc / unidocAllSources :=
+    (JavaUnidoc / unidoc / unidocAllSources).value
+      .map(apiDocSourceFilter(_))
+      .map {
+        _.filterNot { source =>
+          // For JAPI so that some things are excluded from the javadoc because they are internal
+          source.toString.contains("$") || source.toString.contains("packageprivate")
```

Comment:
Could we move these to the `excluded` check in apiSourceDocFilter? No scala files have dollar signs or "packageprivate" in them so it shouldn't affect scala docs.